Common/Sources/Extensions/HyperwalletFee.swift

```diff
@@ -36,6 +36,8 @@ public extension HyperwalletFee {
             description = FeeTypes.flat(flatFee).feeDescription()
         } else if let percentFee = percentFee {
             description = FeeTypes.percent(percentFee).feeDescription()
+        } else {
+            description = FeeTypes.noFee.feeDescription()
         }
```

Comment:
there will never be this case so no need of adding it. If there's no fee present then this func will never be called.

Reply:
Updated